packages/fxa-payments-server/src/routes/Product/SubscriptionUpgrade/PlanUpgradeDetails.tsx

```diff
@@ -34,7 +34,7 @@ export const PlanUpgradeDetails = ({
 
   return (
     <section
-      className={`plan-upgrade-details-component mt-6 mx-4 ${className}`}
+      className={`plan-upgrade-details-component mt-6 tablet:min-w-[18rem] tablet:max-w-xs ${className}`}
```

Comment:
https://tailwindcss.com/docs/width says:

| Class | Properties
|----|----
w-72 | `width: 18rem; /* 288px */`

Not sure if that means we can use an existing `tablet:min-w-72` instead of custom `tablet:min-w-[18rem]`.

Reply:
Hi @pdehaan , I had mentioned above that I had tried `tablet:min-w-72` but when I checked in DevTools it was only being converted for `PlanDetails` and not `PlanUpgradeDetails`. I'll give it another go though.